scripts/node_suite_run.py

```
@@ -48,15 +48,22 @@
 
 def normalize(text: str) -> str:
     out = []
     for line in text.split("\n"):
         if _FRAME.search(line):
             continue
+        if _TEST_LOCATION.match(line):
+            line = "test at <location>"
         line = _DUR_MS.sub(lambda m: "<dur>" + m.group(1), line)
         line = _DUR_CLOCK.sub(lambda m: "<dur:" + m.group(1) + ">", line)
+        line = _TEST_DURATION.sub("(<test-duration>)", line)
+        if line.startswith("ℹ duration_ms "):
```

Ruff (0.15.21)

[warning] String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)